packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -161,37 +168,18 @@ _Props/Attrs_
 -   `key`: string | number | boolean | undefined
 -   `label`: string - display text
 
-### Anatomy
-
-#### `nimble-table-column-enum-text`
+#### Mapping element (placeholder):
 
-```HTML
-<template slot="${x => x.columnInternals.uniqueId}">
-    <slot
-        ${slotted('mappings')}
-        name="mapping">
-    </slot>
-    <span class="header-content">
-        <slot></slot>
-    </span>
-</template>
-```
+The placeholder mapping element will display an empty cell. A group row associated with a placeholder mapping will display the mapping's text. The purpose of the placeholder mapping element is to allow clients to avoid cluttering their table with information that isn't particularly helpful to a user (e.g. that the state of a system is 'idle') while still having a good grouping experience that ensures group rows are not blank.
```

Comment:
A few thoughts:
1. Grouping will still be based on the key values, which could be confusing if multiple placeholder mappings (i.e. with different keys) are used. Of course in that case, different `label`s should be used to mitigate confusion.
2. Initially I thought it might be better to just support `label-hidden` on `nimble-mapping-text` instead of introducing another mapping element, but a text mapping that hides its text is probably weird enough that a new element is warranted.
3. I don't think "placeholder" is the right term. In the table, we've used "placeholder" to mean text we show when a data value is missing. In this case, we're not showing any text, and it's not (necessarily) for missing values. I'd suggest we go with `nimble-mapping-blank` or `nimble-mapping-empty` since this is really for when the client explicitly wants blank cells.

Reply:
I agree that putting `label-hidden` on `nimble-mapping-text` feels weird and that `placeholder` likely isn't the correct term either. I'll update this to `nimble-mapping-blank`.